plugins/module_utils/api.py

```diff
@@ -32,6 +32,7 @@
 FIELD_TEMPLATE = "template"
 FIELD_DATA_RENDERED = "data_rendered"
 FIELD_QUERY_PARAMS = "query_params"
+FIELD_ATTRIBUTES_NAME = "attributes"
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.